src/routes/patterns/components/Alerts.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import Card from 'components/Card';
+
+export default class Alerts extends Component {
+  state = {
+    isDefault: false,
+    isDanger: false
+  };
+
+  showDefault = () => {
```

Comment:
Super rough all around I'm gonna go ahead and trim this PR significantly! just wanted to get some starter files here!